packages/spanner/src/database.js

```diff
@@ -577,6 +593,25 @@ Database.prototype.run = function(query, options, callback) {
  *   .on('end', function() {});
  *
  * //-
+ * // If you need to enforce a specific param type, a types map can be provided.
+ * // This is typically useful if your param value can be null.
```

Comment:
Well, if you were to supply a string, the upstream API is still expecting an INT64 and throws an error, so I don't think that's the correct solution here.